cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -38,13 +38,13 @@ spec:
         - name: config-volume
           mountPath: /etc/google-fluentd/config.d
         env:
-        - name: STACKDRIVER_METADATA_AGENT_URL
-          value: {{ stackdriver_metadata_agent_url }}
         - name: NODE_NAME
           valueFrom:
             fieldRef:
               apiVersion: v1
               fieldPath: spec.nodeName
+        - name: STACKDRIVER_METADATA_AGENT_URL
+          value: http://$(NODE_NAME):8799
```

Comment:
$(NODE_NAME) is not visible here. Please generate the url in configure-helper.sh and substitute it with sed as it is done now. Please make sure to test that it works as intended before submitting.

Reply:
Node name is visible here via Kubernetes bash variable substitution. This is how we're doing it in our manual installation instructions, without the use of bash sed replacing. You can see the configuration as we apply it verbatim here in our public config: https://storage.googleapis.com/stackdriver-gke-beta/configs/logging-agent.yaml 

I can try to find the official documentation for this feature, but it has been tested and validated by our team.